sphire/protocols/protocol_cryolo_picking.py

```diff
@@ -236,6 +240,12 @@ def _validate(self):
                     "website and ~/.config/scipion/scipion.conf must contain "
                     "the '%s' parameter pointing to the downloaded file."
                     % CRYOLO_GENMOD_VAR)
+            elif self.inputModelFrom == INPUT_MODEL_GENERAL_NS:
+                validateMsgs.append(
+                    "The general model for cryolo (negative stain) must be download from Sphire "
+                    "website and ~/.config/scipion/scipion.conf must contain "
```

Comment:
We are hardcoding the path for the config but we should start to use the value from the config object:
pyworkflow.Config.SCIPION_LOCAL_CONFIG